### PR TITLE
collections: reclist cache fix

### LINKED ISSUE #3051: collections: record restricted after insert
How to reproduce
=============

1. Insert new record with `bibupload -i`
2. Run `bibindex -u admin`
3. At that moment the new record is correctly indexed in the `collections` index

```python
In [1]: from invenio.modules.search.searchext.engines.native import \
   ...:                 search_unit_in_idxphrases

In [3]: 358 in search_unit_in_idxphrases("HEP", 'collection', 'e')
Out[3]: True
```

But the record is still **restricted**.

Running

```bash
$ redis-cli flushdb
```

does not fix it.

Only restarting the dev server makes the record visible.

### PR DESCRIPTION
**Pull request by egabancho**
* Fixes `timestamp_verifier` to return the last time the collection
  index was updated. (closes #3051)

Signed-off-by: Esteban J. G. Gabancho <esteban.gabancho@gmail.com>

### CODE REVIEW

File: invenio/modules/collections/cache.py [hunk 1]
```diff
@@ -83,11 +85,16 @@ def cache_filler():
             ])
 
         def timestamp_verifier():
-            from invenio.legacy.dbquery import get_table_update_time
-            return get_table_update_time('collection')
+            return IdxINDEX.query.filter_by(id=11).value(
```
Comment (jirikuncar) [inline invenio/modules/collections/cache.py]: `s/id=11/self._index_id/`

**Comment by jirikuncar** [comment]
:shipit: with `Reviewed-by: me`

**Comment by egabancho** [comment]
:+1:

File: invenio/modules/collections/cache.py [hunk 2]
```diff
@@ -83,11 +85,16 @@ def cache_filler():
             ])
 
         def timestamp_verifier():
-            from invenio.legacy.dbquery import get_table_update_time
-            return get_table_update_time('collection')
+            return IdxINDEX.query.filter_by(id=self._index_id).value(
+                'last_updated').strftime(
+                    "%Y-%m-%d %H:%M:%S")
```
Comment (jirikuncar) [inline invenio/modules/collections/cache.py]: you can now join it with line 89 ;)

Comment (egabancho) [inline invenio/modules/collections/cache.py]: Done!

**Pull request merged**